sacc2/data_types.py

```diff
@@ -0,0 +1,251 @@
+import warnings
+
+from astropy.table import Table
+
+from .utils import Namespace, hide_null_values, null_values
+
+
+required_tags = {
+    "galaxy_shear_xi_plus": ["theta"],
+    "galaxy_shear_xi_minus": ["theta"],
+    "galaxy_shear_xi_plus_imaginary": ["theta"],
+    "galaxy_shear_xi_minus_imaginary": ["theta"],
+    "galaxy_density_w":["theta"] ,
+    "ggl_gamma_t":["theta"] ,
+    "ggl_gamma_x":["theta"] ,
+    "galaxy_shear_ee": ["ell"],
+    "galaxy_shear_bb": ["ell"],
+    "galaxy_shear_eb": ["ell"],
+    "galaxy_density_cl": ["ell"],
+    "ggl_E": ["ell"],
+    "ggl_B": ["ell"],
```

Comment:
Do you mean the ones here: 
https://ccl.readthedocs.io/en/v1.0.0/api/pyccl.correlation.html#pyccl.correlation.correlation
?
We definitely need more than that since we want to measure data types that are always zero in the theory predictions.

Something more consistent would be better than what's there now, though.

Reply:
Resolving for now

Reply:
agreed we need more - we should fix this sooner rather than later since it is a core part of the api and future changes will be painful.